src/events.ts

```diff
@@ -7,6 +7,7 @@ interface RefStudioEvents {
   // Menu actions (Note that these should have undefined payload)
   'refstudio://menu/file/new': undefined;
   'refstudio://menu/file/save': undefined;
+  'refstudio://menu/file/markdown': undefined;
```

Comment:
I wasn't sure what the convention was here. The others menu events are just the "path" in the menu and `Save File as Markdown...` lives directly under the `File` menu and not inside an `Export` submenu, so that is why I chose this name. But I can add `/export/` if we don't need the name of the event to represent the path

Reply:
I see your (fair) point. We can also make that change later.